Mathlib/MeasureTheory/Integral/DivergenceTheorem.lean

```diff
@@ -264,9 +264,8 @@ We represent both faces `x i = a i` and `x i = b i` as the box
 of `f : ℝⁿ⁺¹ → Eⁿ⁺¹` to these faces are given by `f ∘ backFace i` and `f ∘ frontFace i`, where
 `backFace i = Fin.insertNth i (a i)` and `frontFace i = Fin.insertNth i (b i)` are embeddings
 `ℝⁿ → ℝⁿ⁺¹` that take `y : ℝⁿ` and insert `a i` (resp., `b i`) as `i`-th coordinate. -/
-theorem integral_divergence_of_hasFDerivWithinAt_off_countable (hle : a ≤ b)
-    (f : ℝⁿ⁺¹ → Eⁿ⁺¹)
-    (f' : ℝⁿ⁺¹ → ℝⁿ⁺¹ →L[ℝ] Eⁿ⁺¹)
+theorem integral_divergence_of_hasFDerivAt_off_countable (hle : a ≤ b)
```

Comment:
Could you also fix the name of `integral_divergence_of_hasFDerivWithinAt_off_countable_aux₂`?